examples/envs/lung/lung_pid.ipynb

```diff
@@ -0,0 +1,182 @@
+{
```

Comment:
Could we split into three files to mirror the `lung` directory?

e.g., `lung/balloon_lung.ipynb` (removing leading underscore because these aren't "private" files, but public examples)

Reply:
I was thinking about that originally, but wouldn't this create alot of redundant code? What I mean is that all 3 of the notebooks would be using the same scan vs. forloop testing code

Reply:
Yep, but these are examples, so we prefer them to be self-contained. Maintaining examples is in general is a bit of a pain (the flax team continues to spend a lot of time thinking about how to do this well).